packages/fast-development-site-react/app/design-system/index.ts

```diff
@@ -0,0 +1,14 @@
+export enum Direction {
```

Comment:
You've defined this several times - is it not sharable?

Reply:
wasn't sure where to put it initially, now fixed, exported from the fast-application-utilities